scripts/generate_package_diffs_for_flavors.py

```diff
@@ -0,0 +1,78 @@
+import sys
+import pandas as pd
+import subprocess
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from io import StringIO
+
+
+def parse_package_list_file(file_path:Path)->str:
+    parse_package_list_file_command = ["perl","ext/scripts/list_newest_versions/extract_columns_from_package_lisl.pl","--file",str(file_path),"--columns","0,1"]
+    result = subprocess.run(parse_package_list_file_command,stdout=subprocess.PIPE)
+    result.check_returncode()
+    result_string = result.stdout.decode("UTF-8")
+    return result_string
+
+def compare_package_lists(package_list_1:str, package_list_2:str)->pd.DataFrame:
```

Comment:
Maybe for better understanding we could name the package lists:
package_list_new  and package_list_old